File: src/test/resources/mark_java/simplesplit_splitstring.java

class bla {

  java.lang.String call(java.lang.String s) {
    return "";
  }


  int test1() {
    java.lang.String s = "AES/CBC/123";

    bla b = new bla();

    b.call(s);
    return 9;
  }

  int test2_ok() {
      java.lang.String s = "SIV/SIV/123";

      bla b = new bla();

      b.call(s);
      return 9;
  }

}

File: codyze-v2/src/test/resources/mark_cpp/simplesplit_splitstring.cpp
#include <iostream>

class Test {

public:
    int call(string s) {
      return 1;
    }
}


int pass() {
  string s = "AES/CBC/123";

  Test t();

  t.call(s);
}


int test2_ok() {
  string s = "SIV/SIV/123";

  Test t();

  t.call(s);
  return 9;
}